test/test_mountain_array.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from challenges.mountain_array import solution
+
+class TestArray(unittest.TestCase): 
+    def test_(self): 
+        """Testing the solution function to the mountain array challenge
+        """
+
+        self.assertEqual(solution([2,1]), False)
+        self.assertEqual(solution([3,5,5]), False)
+        self.assertEqual(solution([0,3,2,1]), True)
```

Comment:
For boolean assertions like these, consider using `assertTrue` and/or `assertFalse`
```suggestion
        self.assertFalse(solution([3,5,5]))
        self.assertTrue(solution([0,3,2,1]))
```

Reply:
thank you, adjusted.